compiler/test/stdlib/number.test.gr

```diff
@@ -0,0 +1,42 @@
+import Number from "number"
+// add
+assert Number.add(25, 5) == 30
+// sub
+assert Number.sub(25, 5) == 20
+// mul
+assert Number.mul(5, 5) == 25
+// div
+assert Number.div(25, 5) == 5
+// sqrt
+assert Number.sqrt(25) == 5
+assert Number.sqrt(25.0) == 5.0
+assert Number.sqrt(35) == 5.916079783099616
+// min
+assert Number.min(5, 6) == 5
+assert Number.min(6, 5) == 5
+// max
+assert Number.max(6, 5) == 6
+assert Number.max(5, 6) == 6
+// ceil
+assert Number.ceil(-25.5) == -25
+assert Number.ceil(25.5) == 26
+assert Number.ceil(25) == 25
+// floor
+assert Number.floor(-25.5) == -26
+assert Number.floor(25.5) == 25
+assert Number.floor(25) == 25
+// trunc
+assert Number.trunc(-25.5) == -25
+assert Number.trunc(25.5) == 25
+assert Number.trunc(25) == 25
+// round
+assert Number.round(-25.5) == -26
```

Comment:
I think it would be good to have some tests for rational numbers as well for each of these, i.e. `assert Number.round(22/7) == 3`.